engage/app/templates/calendar.hbs

```diff
@@ -1,7 +1,7 @@
 <div class="row">
   <div class="col-md-8 main-left">
     <h2 class="page-header">Calendar</h2>
-    <div class="media list-group-item engage-list-group-item calendar-item">
+    <div class="media content-subject-item">
```

Comment:
`content-subject-item` doesn't mean anything to me... why don't we call this `engage-media` to keep the pattern we've done with `engage-list-group-item` ie, BS class + our modifications